src/CookiePlugin.php

```diff
@@ -109,11 +109,12 @@ private function createCookie(RequestInterface $request, $setCookie)
 
             switch (strtolower($key)) {
                 case 'expires':
-                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value);
+                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value)->getTimestamp()
```

Comment:
I don't think the expires header should return an integer as well. It is defined as datetime.